roles/firefox-client/templates/firefox-extension-update.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+#
+# Datei wird über Ansible bereitgestellt und muss auch dort bearbeitet werden.
+#
+
+if cd /usr/local/share/ansible_downloads; then
+downloads=({% for extension in firefox_extensions %}
+"`wget -N {{ extension.url }}`"{% endfor %}); fi
```

Comment:
Ich mag die Backtick Notation  für Befehl ausführen nicht so 
was hälst du von $(wget -N ... )?

Was macht das Skript, wenn mehrere Dateien heruntergeladen werden? steht da nicht 
"downloadfile1""downloadfile2"

Was dann 
"downloadfile1downloadfile2" wird

also ich glaube da fehlt ein Leerzeichen.

Außerdem bin ich mir nicht sicher ob der Server auf der anderen Seite wget nicht einen anderen Filenamen geben kann, das kann dann übel werden, wenn da Sonderzeichen dinn sind.

Reply:
Was hälst du von 
```
$(wget -N {{ extension.url }} -O {{ extension.name }})
```

Reply:
Das Skript war schon in einer vollständigen Überarbeitung, da es mir noch nicht gefallen hat und auch noch keine Fehlermeldungen gesendet hat.
Die Backtick Notation war schon rausgeflogen.
Die Option -O führt dazu, dass das Änderungsdatum auf das aktuelle Datum gesetzt wird, da praktisch eine redirection verwendet wird.
Das Sonderzeichenproblem ist natürlich valide. Ich denke aber im Moment haben wir keine andere Wahl.

Reply:
Was hältst du von der dem neusten commit für das Skript?